plugins/content/config/model.schema

```diff
@@ -51,17 +51,15 @@
           "isSetting": false,
           "inputType": "Checkbox",
           "validators": [],
-          "title": "Enable accessibility?",
-          "help": "If enabled, accessibility features will be enabled in this course"
```

Comment:
Is there any reason to remove the help?

Reply:
Yes, because it's not particularly helpful in that it doesn't add any more information than you'd glean from reading the title!

Reply:
It should probably be changed to be more useful rather than being removed such as. `Enables accessibility features in this course, such as extra tags for screen readers` 

Remember the users of the AT are often not as experienced as the framework users and being explicit even when it is super obvious to us can help.

Reply:
Changing the title to 'Enable accessibility features?' would go some way towards making it clearer... but I'm not sure 'extra tags for screen readers' adds much in the way of useful info. 

I'm sure there is something better that can be written, I just don't feel qualified to write it!

How about I amend the title as suggested above and we raise another PR for 'improvements to help text'?

One other possibility would be to include a link to https://github.com/adaptlearning/adapt_framework/wiki/Accessibility in the help text

Reply:
Ok sure, that issue would be best on the framework repo I think.